flatpak-json2yaml.py

```diff
@@ -89,8 +93,16 @@ def main():
 
         yaml_data = json_to_yaml(h.read())
 
-        with open(out_file, 'wb') as out:
-            out.write(yaml_data)
+        if args.force:
+            flags = O_CREAT|O_WRONLY|O_CLOEXEC
+        else:
+            flags = O_CREAT|O_EXCL|O_WRONLY|O_CLOEXEC
+
+        fd = os.open(out_file, flags)
```

Comment:
I would have kept using context managers here instead of try/finally

Reply:
sure. it's not possible to do with `os.open` though.